rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -41,6 +41,8 @@ For more information on region availabilities, see xref:../rosa_architecture/ros
 
 * **Log linking is enabled by default.** Beginning with {product-title} 4.15, log linking is enabled by default. Log linking gives you access to the container logs for your pods.
 
+* **Node management improvements** Now, you can perform specific tasks to make clusters more efficient. You can cordon, uncordon, and drain a specific node. For more information, see xref:../nodes/nodes/nodes-nodes-working[Working with nodes].
```

Comment:
Add ".adoc" to xref path:

`xref:../nodes/nodes/nodes-nodes-working.adoc[Working with nodes]`